app/build.gradle.kts

```diff
@@ -108,3 +108,9 @@ tasks.register<Delete>("deleteUnsupportedPlayTranslations") {
         "src/main/play/listings/ta/",
     )
 }
+
+project.afterEvaluate {
+    tasks.named("mergeReleaseJniLibFolders") {
+        dependsOn(":syncthing:buildNative")
+    }
+}
```

Comment:
Newer versions of Gradle check for [implicit dependencies between tasks](https://docs.gradle.org/8.5/userguide/validation_problems.html#implicit_dependency).

With an implicit dependency the build fails as follows:

<img width="1887" alt="Screenshot 2024-01-05 at 22 02 06-updated" src="https://github.com/syncthing/syncthing-android/assets/6529875/c6d9f07d-c70f-49ce-9e0f-8b4b01c9162e">

The above change makes the dependency explicit.